src/components/Toggle/README.md

```diff
@@ -0,0 +1,40 @@
+Toggle
```

Comment:
No need for this title here. It will duplicate like so:
<img width="390" alt="screen shot 2018-01-19 at 16 57 38" src="https://user-images.githubusercontent.com/2573602/35166711-f3f91f5a-fd39-11e7-8130-a0ee52d6679b.png">